src/internal/config/config.proto

```diff
@@ -95,6 +95,10 @@ message Context {
     // we ensure this is the same as what the cluster reports back, to prevent
     // us from connecting to the wrong cluster.
     string cluster_deployment_id = 11 [(gogoproto.customname) = "ClusterDeploymentID"];
+
+    // A boolean that records whether the context points at an enterprise server,
```

Comment:
minor nit: IIUC, in the "unified" case (pachd and enterprise are on the same server), this should be `false`, since the app for that pod will be `pachd`. I wonder if it would make sense to say e.g. `if true, this context points to a standalone Pachyderm enterprise server, rather than a regular pachd pod` or something along those lines, to maybe make the default clearer?